drivers/SmartThings/matter-switch/fingerprints.yml

```diff
@@ -59,6 +59,27 @@ matterManufacturer:
     vendorId: 0x115a
     productId: 0x44
     deviceProfileName: light-color-level-2700K-6500K
+#SONOFF
+  - id: "SONOFF MINIR4M"
+    deviceLabel: Smart Plug-in Unit
+    vendorId: 0x1321
+    productId: 0x0002
+    deviceProfileName: plug-binary
```

Comment:
Hi @lisongjun12 - the fingerprint for MiniR4m is not needed since it is already in our repo. Please remove these lines:
- id: “SONOFF MINIR4M”
    deviceLabel: Smart Plug-in Unit
    vendorId: 0x1321
    productId: 0x0002
    deviceProfileName: plug-binary

Thank you!